### ISSUE TITLE
Autoscroll doesn't work when web screen reader is on 

### ISSUE THREAD

**Issue by pooja96km** [flutter/flutter#122686]
When screen readers are activated, the pages do not always auto scroll to the elements beyond the visible area of the browser. This happens even when the reader announces the semantics for the element that is outside the currently visible area.
